openshift/message-board/README.asciidoc

```diff
@@ -65,4 +70,19 @@ http://web-message-board.[YOUR-IP].nip.io/account-service/user
 #### without proxy
 ----
 http://account-service-message-board.[YOUR-IP].nip.io/account-service/user
-----
\ No newline at end of file
+----
+
+## Known Issues
+
+### NodeJs module Angular cli 6.1.x
+It seems there is an issue on angular-cli versions 6.1.0 and 6.1.1.
+It might need to downgrade angular-cli version:
+----
+sudo -s npm uninstall -g @angular/cli
+sudo -s npm install -g @angular/cli@6.0.8
+----
+For more information see https://github.com/angular/angular-cli/issues/11661[Issue Page].
+
+### MiniShift CDK v3.5.0-1
+Actually there is an issue with MiniShift v1.20.0+1bd6d5c, CDK v3.5.0-1.
+The persistent volume for Infinispan server pod is mounted as root user.
```

Comment:
Is this the current version? Is it a real issue for the demo?

How do I fix it?

You're not talking about MiniShift before so it comes a bit out of the blue.

Reply:
Yes it is in the current (last at the moment) version. Actually I don't know how to fix it, thus I'm using the Openshift Oring provided by Fedora 28, where the issue does not occur.